pkg/storaged/details.js

```diff
@@ -1628,22 +1637,29 @@ define([
 
             function make_pvol(pvol) {
                 var block = client.blocks[pvol.path];
-                var actions = [
-                    { action: "pvol_remove", title: _("Remove") },
-                    { action: "pvol_empty",  title: _("Empty") }
-                ];
+                var action = null;
+                var excuse = null;
+                if (pvols.length == 1) {
+                    excuse = _("The last physical volume of a volume group can not be removed.");
```

Comment:
There are multiple "can not"s in the storage code already...  Should we be consistent until we change them all?